### PR TITLE
fix(replays): Add check for PII scrubbed replay-id tag

### PR DESCRIPTION
**Pull request by cmanallen**
Old SDKs will submit the replayId in the `tags`.  This leaves the `replayId` vulnerable to PII scrubbing.  We verify the UUID is valid before submitting to snuba.

### CODE REVIEW

File: src/sentry/tasks/post_process.py [hunk 1]
```diff
@@ -891,6 +892,12 @@ def _get_replay_id(event):
     if not replay_id:
         return
 
+    # Validate the UUID.
+    try:
+        uuid.UUID(replay_id)
+    except (ValueError, TypeError):
```
Comment (JoshFerge) [inline src/sentry/tasks/post_process.py:898]: should we increment a metric here?

**Review by JoshFerge — APPROVED** [review]
💚

File: src/sentry/tasks/post_process.py (hunk 1 above)
Comment (cmanallen) [reply src/sentry/tasks/post_process.py:898]: I was thinking about this earlier but couldn't come up with a use case for why I would want to know it.  Do you have anything in mind?

Comment (JoshFerge) [reply src/sentry/tasks/post_process.py:898]: how often this happens so we can prioritize a solution if needed.

**Pull request merged**